src/templates/kit/screens/ReviewScreen.js

```diff
@@ -18,8 +23,35 @@ function Review({
     next(data)
   }, [data, next])
 
+  const containerRef = useRef()
+  const prevNextRef = useRef()
+
+  useEffect(() => {
+    if (containerRef.current) {
+      containerRef.current.focus()
+    }
+  }, [])
```

Comment:
Should `container.Ref` be a dependency here?

Reply:
ref containers are always the same, so I don’t think it is ever needed?